src/backend/db/categories.js

```diff
@@ -8,20 +11,30 @@ import { v4 as uuid } from "uuid";
 export const categories = [
   {
     _id: uuid(),
-    categoryName: "Computer Programming",
-    description:
-      "Computer programming is the process of designing and building an executable computer program to accomplish a specific computing",
+    categoryName: "Movies",
+    icon:"fas fa-video",
+    image:"https://res.cloudinary.com/sushil-mod/image/upload/v1654541751/jediTube/movies.webp",
+    
+  },
+  {
+    _id: uuid(),
+    categoryName: "Animated",
+    icon:"fas fa-radiation",
+    image:"https://res.cloudinary.com/sushil-mod/image/upload/v1654541739/jediTube/animation.jpg",
+   
   },
   {
     _id: uuid(),
-    categoryName: "Frontend Development",
-    description:
-      "Front-end web development, also known as client-side development is the practice of producing HTML, CSS and JavaScript for a website or Web Application",
+    categoryName: "Lego",
+    icon:"fas fa-robot",
+    image:"https://res.cloudinary.com/sushil-mod/image/upload/v1654542195/jediTube/lego.jpg",
+    
   },
   {
     _id: uuid(),
-    categoryName: "Backend Development",
-    description:
-      "Backend Development is also known as server-side development. It is everything that the users don't see and contains behind-the-scenes activities that occur when performing any action on a website.",
+    categoryName: "Games",
+    icon:"fas fa-gamepad",
+    image:"https://res.cloudinary.com/sushil-mod/image/upload/v1654541729/jediTube/games.jpg",
```

Comment:
Didn't knew about cloudinary image hosting service. New thing you tried out here as well

Reply:
just hosting image and using url